.eslintignore

```diff
@@ -0,0 +1,3 @@
+packages/babel-polyfill-udemy/helpers.js
```

Comment:
I renamed `babel-environment-udemy`, and `babel-preset-udemy` as `babel-polyfill-udemy` and `babel-preset-udemy-website` respectively. But I am still debating which option would be the best:

1. `babel-polyfill-udemy` and `babel-preset-udemy-website`
1. `babel-polyfill-udemy-basics` and `babel-preset-udemy-basics`
1. `babel-polyfill-udemy-website` and `babel-preset-udemy-website`

@udemy/team-f opinions?